cura/CuraApplication.py

```diff
@@ -1028,6 +1030,8 @@ def registerObjects(self, engine):
         qmlRegisterSingletonType(SimpleModeSettingsManager, "Cura", 1, 0, "SimpleModeSettingsManager", self.getSimpleModeSettingsManager)
         qmlRegisterSingletonType(MachineActionManager.MachineActionManager, "Cura", 1, 0, "MachineActionManager", self.getMachineActionManager)
 
+        qmlRegisterType(NetworkingUtil, "Cura", 1, 0, "NetworkingUtil")
```

Comment:
```suggestion
        qmlRegisterType(NetworkingUtil, "Cura", 1, 5, "NetworkingUtil")
```
This number is supposed to indicate an API of sorts. As it was added later, we should let it have a higher number.

Reply:
Done in 40d26bbb6ecff89540bcc2ccbe2797440339df4e